.env.testing

```diff
@@ -1,5 +1,6 @@
 ES_URL=http://localhost:19200
 NEO4J_URL=bolt://:@localhost:17687
+DIALOGFLOW_ADAPTER=http://www.localhostdrolf.de
```

Comment:
Some url that tells you what is wrong would be nice here imo. i.e. `tobito.should.be.mocked`